### PR TITLE
chore: Upgrade to typescript 3.7.2

### PR DESCRIPTION
**Pull request by nulltoken**
**Checklist**

- [ ] Tests added / updated
- [ ] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -114,7 +114,7 @@
     "rollup-plugin-commonjs": "^10.0.2",
     "rollup-plugin-node-resolve": "^5.2.0",
     "rollup-plugin-terser": "^5.1.1",
-    "rollup-plugin-typescript2": "^0.25.1",
+    "rollup-plugin-typescript2": "^0.25.2",
```
Comment (nulltoken) [inline package.json]: @P0lip Looks like version 0.25.1 disappeared from npm registry...

image


...weird...

Comment (P0lip) [reply package.json]: Interesting. Looks like author marked it as deprecated.
https://github.com/ezolenko/rollup-plugin-typescript2/releases
Let's update to 0.25.2 then.

**Comment by nulltoken** [comment]
@P0lip No longer a WIP. Ready for review.

**Comment by nulltoken** [comment]
> Resolve conflicts and we are good to go.

@P0lip Done

**Pull request merged**